docker_files/recipes/python/meta/run.yaml

```diff
@@ -0,0 +1,8 @@
+runtime: native
+config_set:
+  shell:
```

Comment:
Do you think naming it `python` would be better? I find it more intuitive to use:
```
$ capstan run demo --boot python
```